proxy/src/org/phoenixbioinformatics/proxy/Proxy.java

```diff
@@ -748,15 +749,24 @@ private Boolean authorizeProxyRequest(String secretKey, String partnerId,
                   + " needs to subscribe to see paid content " + fullUri
                   + " at partner " + partnerId);
       StringBuilder uriBuilder = new StringBuilder(uiUri);
-
-      try {
-        String meter =
-          ApiService.checkMeteringLimit(remoteIp, partnerId, fullUri);
-
+      
```

Comment:
Just to double-check, does the "free" quota logic remain the same? i.e. each user still has some free page hits on a per IP basis

Reply:
An important note: MorphoBank and GIRI and still using the same part of the logic, so we have to put back the old logic for them!